src/data/useYtelseSammendrag.ts

```diff
@@ -1,16 +1,17 @@
 import { useAppStore } from './store/AppStore';
+import { useYtelseSammendragContext } from './store/YtelseSamendrag';
 import { YtelseSammendrag } from '../util/helseSpionTypes';
 import dayjs from 'dayjs';
 import env from '../Environment';
 
 
 const useYtelseSammendrag = (): any => {
-  const { setYtelsesammendrag } = useAppStore();
+  const { setYtelsesammendrag } = useYtelseSammendragContext();
```

Comment:
Navnestandarden for custom react hooks er å prefikse med use - altså ikke bruker postfix Context.

Reply:
Har en annen hook som heter useYtelseSammendrag

Reply:
Alle typer hooks starter med use, uavhengig om de har noe med context å gjøre eller ikke.